tests/unit/test_certificates_integration.py

```diff
@@ -50,7 +52,7 @@ def test_cert_relation(
     get_secret_mock = MagicMock()
     monkeypatch.setattr(ops.model.Model, "get_secret", get_secret_mock)
 
-    harness.update_config({"external-hostname": "igress-internal"})
+    harness.update_config({"external-hostname": "igress-internal", "gateway-class": "cilium"})
```

Comment:
Could we use the defined constant here?

Reply:
Also, I think `igress-internal` should be `ingress-internal`.

Reply:
moved the constant over to this file.